apps/web/src/components/Composer/Actions/Attachment.tsx

```diff
@@ -91,99 +90,118 @@ const Attachment: FC = () => {
     }
   };
 
+  const preventDefaultEvent = (e: Event) => {
+    e.preventDefault();
+  };
```

Comment:
we have a global util called `stopEventPropagation.ts`

Reply:
Sure, will use that.

Reply:
This `stopEventPropagation.ts` utility cannot be used as I don't want to stop the event propagation, instead of that I want to prevent the default event itself.